src/Microsoft.HttpRepl/Commands/SetQueryStringCommand.cs

```diff
@@ -0,0 +1,91 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.HttpRepl.Resources;
+using Microsoft.HttpRepl.Suggestions;
+using Microsoft.HttpRepl.Telemetry;
+using Microsoft.HttpRepl.Telemetry.Events;
+using Microsoft.Repl;
+using Microsoft.Repl.Commanding;
+using Microsoft.Repl.ConsoleHandling;
+using Microsoft.Repl.Parsing;
+
+namespace Microsoft.HttpRepl.Commands
+{
+    public class SetQueryStringCommand : ICommand<HttpState, ICoreParseResult>
+    {
+        private const string CommandName = "set";
+        private const string SubCommand = "queryString";
+
+        private readonly ITelemetry _telemetry;
+
+        public string Name => "setQueryString";
+        public static string Description => Strings.SetQueryStringCommand_HelpDetails;
+
+        public SetQueryStringCommand(ITelemetry telemetry)
+        {
+            _telemetry = telemetry;
+        }
+
+        public bool? CanHandle(IShellState shellState, HttpState programState, ICoreParseResult parseResult)
+        {
+            return parseResult.ContainsAtLeast(minimumLength: 3, CommandName, SubCommand)
+                ? (bool?)true
```

Comment:
Interesting. What version of VS are you using?

Reply:
Version 16.8.2.
I think is the latest.

Reply:
Hm. Yeah I can repro that too. 16.8.2 doesn't like it, but the 16.9 previews I have on my box do. Not sure why they'd be different here. Oh well. You can leave it how it works in 16.8.2 since that's the non-preview release.